src/js/components/agency/visualizations/StatusOfFundsChart.jsx

```diff
@@ -91,7 +94,7 @@ const StatusOfFundsChart = ({
         }, 50);
         window.addEventListener('resize', handleResize);
         return () => window.removeEventListener('resize', handleResize);
-    }, []);
```

Comment:
same thing here, the windowWidth dependency shouldn't be necessary and could cause unnecessary event binding and removal